README.md

```diff
@@ -1,12 +1,33 @@
 # ALBA
+This is the rust library of the *Telescope* protocol proposed in the paper [Approximate Lower Bound Arguments](https://iohk.io/en/research/library/papers/approximate-lower-bound-arguments/), May 2024, Eurocrypt'24 by _Pyrros Chaidos, Prof Aggelos Kiayias, Leonid Reyzin, Anatoliy Zinovyev_.
 
-Implementation of _Approximate Lower Bound Arguments_ from the [paper](https://iohk.io/en/research/library/papers/approximate-lower-bound-arguments/) published by IOG Research:
+Alba is a generic protocol to prove succinctly a lower bound of the size of a *potentially* weighted set.
+Say we have a set $S_p$ of size $|S_p| \geq n_p$, and a lower bound $n_f < n_p$ of it we want to prove.
+Alba gives us a method to generate a proof of knowledge of this bound by finding the smallest subset of $S_p$ of size $u$ to convince a verifier.
```

Comment:
finding `the smallest` vs. `a smallest` - if we use a definite article this assumes that the set is unique. Also, is `u` really the cardinality of a smallest subset - in other words, can't the algorithm find a set that has a proper subset that will also convince a verifier. We should be precise here.